### DISCUSSION TITLE
How long does it take on average to receive a reply for a support for a ticket

### DISCUSSION THREAD

**Discussion by BrennerLittle** [community/community#23844] [New to GitHub]
<p>I am trying to contact support about deleting one of my repositories that I am having trouble deleting. I created a support ticket about how long does it take for it to be answered on average? Thank you.</p>

**Comment by BrennerLittle** [comment, marked as answer]
<p>I got a response quite quickly in about 12 hours.</p>

**Reply by crizmo** [reply]
damn me replying on an old post but 
its been 22 days now and i still haven't gotten any response

**Reply by ghost** [reply]
@crizmo how about now, any reply from their side?

**Reply by crizmo** [reply]
Yes I got their response,  there was some issue with my pro plan thingy 
It's fixed now

**Reply by mhfdania** [reply]
Hi, I created a support ticket as well about the same issue, about how long did it take for it to be answered please ? Thank you.
